webapp/src/components/rhs/rhs_run_list.tsx

```diff
@@ -226,6 +245,32 @@ const StyledLoadingSpinner = styled(LoadingSpinner)`
     align-self: center;
 `;
 
+const TitleContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+`;
+
+const VerticalLine = styled.div`
+    opacity: 0.16;
+    border: 1px solid var(--center-channel-color);
+    height: 24px;
+`;
+
+const ChannelNameText = styled.div`
+    color: rgba(var(--center-channel-color-rgb), 0.56);
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 20px;
```

Comment:
Long channel names case:

![Screenshot 2022-12-01 at 10 07 28](https://user-images.githubusercontent.com/4096774/205012611-5ed8064d-61b9-4f36-8e94-cd10b885281e.png)


```suggestion
    line-height: 20px;
    overflow: hidden;
    text-overflow: ellipsis;
```

Additionaly, TitleContainer would need some min-width